django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -157,4 +157,13 @@
     window.showAddAnotherPopup = showRelatedObjectPopup;
     window.dismissAddAnotherPopup = dismissAddRelatedObjectPopup;
 
-})();
+    $(document).ready(function() {
+        $("a[data-popup_opener]").each(function() {
```

Comment:
This can be simplified to

```javascript
$(function(){
    $("a[data-popup_opener]").click(function(event){
        event.preventDefault();
        opener.dismissRelatedLookupPopup(window, $(this).data("popup_opener"));
    });
});
```